dss/config.py

```diff
@@ -388,6 +389,15 @@ def get_notification_email() -> str:
 
         return Config._NOTIFICATION_SENDER_EMAIL
 
+    @staticmethod
+    def get_admin_user_emails() -> typing.List[str]:
+        envvar = "ADMIN_USER_EMAILS"
+        if envvar not in os.environ:
```

Comment:
there is already a method that provides this functionality... `_get_required_envvar`

Reply:
Fixed `get_admin_user_emails()` to use `_get_required_envvar` in a39d032

Fixed `get_notification_email()` to use `_get_required_envvar` in 7ebad72

Thanks for the tip!